proposals/004-enable-importing-css-modules-via-filename-convention-used-in-common-frameworks.md

```diff
@@ -0,0 +1,115 @@
+# Enable importing CSS Modules via filename convention used in common frameworks
```

Comment:
Think we discussed using the word projects rather than frameworks? i.e. create-react-app vs. react

Reply:
it's a bit of a fuzzy line what is a library, framework, project.

I think it's probably OK given the subject of the proposal to use framework here.

Many people argue that React is a library not a framework 🤷‍♂️

Reply:
also the convention for .module.css is part of create-react-app (the build system), not React itself (the view library) but I don't mind the wording staying as-is